src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -0,0 +1,250 @@
+import QuestionTooltip from 'app/components/questionTooltip';
+import {t, tct} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {DataCategory, Organization} from 'app/types';
+
+import Chart from './chart';
+import {OrganizationUsageStats} from './types';
+import {formatUsageWithUnits} from './utils';
+
+type Props = {
+  organization: Organization;
+
+  dataCategory: DataCategory;
+  dataCategoryName: string;
+  onChangeDataCategory: (dataCategory: DataCategory) => void;
+} & AsyncComponent['props'];
+
+type State = {
+  orgStats: OrganizationUsageStats;
+} & AsyncComponent['state'];
+
+class UsageStatsOrganization extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization} = this.props;
+
+    // TODO(org-stats): Allow user to pick date range
+    const fourWeeksAgo = moment().subtract(31, 'days').unix();
+    const today = moment().unix();
+
+    return [
+      [
+        'orgStats',
+        `/organizations/${organization.slug}/stats_v2/`,
+        {
+          query: {
+            start: fourWeeksAgo,
+            end: today,
+            interval: '1d',
+          },
+        },
+      ],
+    ];
+  }
+
+  /**
+   * Ignore this hard-coded method. API response is being changed so this will
+   * be amended in a few days.
+   */
+  get formattedOrgStats(): {
+    stats: any[];
+    total: string;
+    accepted: string;
+    dropped: string;
+    filtered: string;
+    overQuota: string;
+  } {
+    const {dataCategory} = this.state;
+
+    const formatOptions = {
+      isAbbreviated: dataCategory !== DataCategory.ATTACHMENTS,
+      useUnitScaling: dataCategory === DataCategory.ATTACHMENTS,
+    };
+
+    return {
+      stats: [],
+      total: formatUsageWithUnits(0, dataCategory, formatOptions),
+      accepted: formatUsageWithUnits(0, dataCategory, formatOptions),
+      dropped: formatUsageWithUnits(0, dataCategory, formatOptions),
+      filtered: formatUsageWithUnits(0, dataCategory, formatOptions),
+      overQuota: formatUsageWithUnits(0, dataCategory, formatOptions),
+    };
+  }
+
+  renderCards() {
+    const {dataCategory, dataCategoryName} = this.props;
+    const {total, accepted, overQuota: overQuota, filtered} = this.formattedOrgStats;
+
+    const cardData = [
+      {
+        title: tct('Total [dataCategory]', {dataCategory: dataCategoryName}),
+        value: total,
+      },
+      {
+        title: t('Accepted'),
+        value: accepted,
+      },
+      {
+        title: t('Filtered'),
+        description: tct(
+          'Filtered [dataCategory] were blocked due to your inbound data filter rules',
+          {dataCategory}
+        ),
+        value: filtered,
+      },
+      {
+        title: t('Rate-limited'),
+        description: tct(
+          "Rate-limited [dataCategory] were discarded due to usage exceeding your plan's quota.",
```

Comment:
do OSS organizations have plans and quotas? also, events can be dropped for other reasons, primarily project key rate limits.